### PR TITLE
Change 309 error mapping

### PR DESCRIPTION
**Pull request by phieronymus**
Updates https://github.com/univapaycast/univapay-form-checkout/issues/923

### CODE REVIEW

File: src/errors/parser.ts [hunk 1]
```diff
@@ -18,7 +18,7 @@ export const getProcessingErrorCodeByStatus = (status: number): string => {
         306: ResponseErrorCode.CardRejected,
         307: ResponseErrorCode.CardInvalid,
         308: ResponseErrorCode.ChargeInvalidData,
-        309: ResponseErrorCode.ExpirationDateError,
```
Comment (gdp88) [inline src/errors/parser.ts:21]: remove `ExpirationDateError` from `ResponseErrorCode` enum if it's not used somewhere else?

**Review by gdp88 — APPROVED** [review]

**Comment by gdp88** [comment]
just a minor comment..lgtm!

File: src/errors/parser.ts (hunk 1 above)
Comment (phieronymus) [reply src/errors/parser.ts:21]: This is an error from the API. This should not be removed.

**Comment by gdp88** [comment]
merging!

**Pull request merged**